### ISSUE TITLE
User check the state of your expenses

### ISSUE THREAD

**Issue by IvelinYR** [IvelinYR/Scrum#11]

**Comment by mgenov** [comment]
Is this related with some report? 

I'm not sure that I understand it correctly but as It looks like a way for users to see daily and monthly reports of expenses.

**Comment by IvelinYR** [comment]
Yes user can see his daily and monthly reports , and he will see if have a mistake in reports.

**Comment by mgenov** [comment]
So he should be able to revert/delete expenses record ?

If so then something like: "manually correct entered expenses records" sounds more reasonable.
